src/os/bluestore/BlueStore.h

```diff
@@ -2164,8 +2164,8 @@ class BlueStore : public ObjectStore,
   bool collections_had_errors = false;
   std::map<coll_t,CollectionRef> new_coll_map;
 
+  mempool::bluestore_cache_buffer::vector<BufferCacheShard*> buffer_cache_shards;
   std::vector<OnodeCacheShard*> onode_cache_shards;
```

Comment:
Add this to mempool (bluestore_cache_meta?) as well?

Reply:
sorry had some stuff come up, makes sense, i have amended the PR, thanks!